client/layout/index.jsx

```diff
@@ -164,7 +164,8 @@ class Layout extends Component {
 				config.isEnabled( 'woocommerce/onboarding-oauth' ) &&
 				isWooOAuth2Client( this.props.oauth2Client ) &&
 				this.props.wccomFrom,
-			'is-nav-unification': isAutomatticTeamMember( this.props.teams ),
+			'is-nav-unification':
+				isAutomatticTeamMember( this.props.teams ) && config.isEnabled( 'nav-unification' ),
```

Comment:
What's the value of `isAutomatticTeamMember` here? Considering that this is only accessible on testing or development environments, and `stage` environments which are accessible only by proxied users (who are a12s anyway), do we really need that check?

Reply:
`isAutomatticTeamMember` is true / false depending on this wpcom function `get_current_user_teams` which actually uses `is_automattician( $this->current_user->ID )` condition.

Reply:
Using the Proxied condition in all environments (Calypso, Simple, Atomic) was an attempt that had only one fault. Atomic doesn't have a `is_supports_session()` check. We had to ditch that so that HEs on support session experience the customers variant. In case you have an idea that we have missed, please let's discuss it!